lib/uffizzi/cli/config.rb

```diff
@@ -17,40 +40,32 @@ def run(command, property, value)
         handle_set_command(property, value)
       when 'delete'
         handle_delete_command(property)
-      else
-        puts "#{command} is not a uffizzi config command"
       end
     end
 
-    private
-
     def handle_list_command
       ConfigFile.list
     end
 
     def handle_get_command(property)
-      if property.nil?
-        puts 'No property provided'
-        return
-      end
+      return Uffizzi.ui.say('No property provided') if property.nil?
+
       option = ConfigFile.read_option(property.to_sym)
-      puts option unless option.nil?
+      Uffizzi.ui.say(option) unless option.nil?
     end
 
     def handle_set_command(property, value)
       if property.nil? || value.nil?
-        puts 'No property provided' if property.nil?
-        puts 'No value provided' if value.nil?
+        Uffizzi.ui.say('No property provided') if property.nil?
+        Uffizzi.ui.say('No value provided') if value.nil?
         return
       end
```

Comment:
The conditions are repeated two times. Just this would be enough:

```
return Uffizzi.ui.say('No property provided') if property.nil?
return Uffizzi.ui.say('No value provided') if value.nil?
```

Reply:
there is no need in these checks since config was reworked as a subcommand
I just forgot to remove them